pom.xml

```diff
@@ -66,8 +66,8 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.375.x</artifactId>
-                <version>2198.v39c76fc308ca</version>
+                <artifactId>bom-2.387.x</artifactId>
+                <version>2550.v4ec98c0b_676d</version>
```

Comment:
Added this as the plug-in version was 2.387 (also needed for latest job-dsl I think, at least from the console logs).